docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.3"
```

Comment:
Let's remove this file. Doesn't seem to be relevant for this PR.

Reply:
I added these files to assist in automated testing. But, if you want them gone, I will remove them. If it's ok with you, I will add them to .gitignore so I can use them locally